lib/ex_machina/ecto.ex

```diff
@@ -75,6 +83,25 @@ defmodule ExMachina.Ecto do
   end
 
   @doc """
+  Works like `params_for/2` but the keys of returned map are strings (instead
+  of atoms). This makes the function useful to be used in controller tests for
+  Phoenix applications where params of this form are expected.
```

Comment:
What do you think about "params of this form" -> "where params with string keys" to make it more explicit?